source/linux/Foundational_Components/U-Boot/UG-General-Info.rst

```diff
@@ -1789,197 +1976,228 @@ to be loaded from the same media as the kernel, and from the same relative path.
-            │                                      │
-            │       ROM Boot parameter table       │
-            │    + Extended boot info (3.5 KB)     │
-            │                                      │
-            └──────────────────────────────────────┘0x43c3ffff
-
-    .. ifconfig:: CONFIG_part_variant in ('J722S')
-
-        .. code-block:: console
-
-            ┌──────────────────────────────────────┐0x43c00000
-            │                                      │
-            │                                      │
-            │               SPL IMAGE              │
-            │            (excluding BSS)           │
-            │            (0x6ce00 B  Max)          │
-            │                                      │
-            ├──────────────────────────────────────┤0x43C6CE00
-            │              EMPTY (0x50 B)          │
-            │                                      │
-            ├──────────────────────────────────────┤0x43C6CE50
-            │                                      │
-            │                                      │
-            │          STACK (0x5000 B Max)        │
-            │                                      │
-            │                                      │
-            ├──────────────────────────────────────┤0x43C71E50
-            │       Global Data (0x1AC B Max)      │
-            │                 (+0x4)               │
-            │                                      │
-            ├──────────────────────────────────────┤0x43C72000
-            │                                      │
-            │            HEAP (0x9000 B Max)       │
-            |                                      |
-            ├──────────────────────────────────────┤0x43C7B000
-            │                                      │
-            │            SPL BSS (0x3000 B)        │
-            │                                      │
-            ├──────────────────────────────────────┤0x43C7E000
-            │                                      │
-            │       ROM Boot parameter table       │
-            │    + Extended boot info (3.5 KB)     │
-            │                                      │
-            └──────────────────────────────────────┘0x43C7F290
+ .. code-block:: console
+
+                ┌         ┌────────────────────┐ 0x7081_8000
+                │         │   Debug Buffers    │
+                │         ├────────────────────┤ 0x7081_6000
+                │         │  TIFS ->  A53 IPC  │
+                │         ├────────────────────┤ 0x7081_5000
+                │         │  A53  -> TIFS IPC  │
+                │         ├────────────────────┤ 0x7081_4000
+                │         │    *Free Space*    │
+                │      ┌  ├────────────────────┤ 0x7081_0000
+                │      │  │ Translation Table  │
+                │      │  ├────────────────────┤ 0x7080_D000
+    MSRAM (96k) │      │  │        BSS         │
```

Comment:
Move labels to the right side of the blocks so indentation is more obvious

Reply:
an yeah that won't be an issue